app/helpers/feed_helper.rb

```diff
@@ -4,6 +4,12 @@ def atom_feed_url_for(resource)
   end
 
   def documents_as_feed_entries(documents, builder, feed_updated_timestamp = Time.current)
+    # This is to support cases where the documents feed has been provided by
+    # another service (e.g. Rummager) rather than from Whitehall via ActiveRecord.
+    if documents.is_a?(GdsApi::Response) && documents['results']
+      documents = documents['results']
```

Comment:
Could we avoid this case if we returned the results in the topical_events_controller.rb `fetch_related_documents`?

Reply:
Yeah good spot, thanks!